nima/tests/integration/webserver/access-log/src/test/java/io/helidon/nima/tests/integration/server/accesslog/AccessLogTest.java

```diff
@@ -85,17 +83,45 @@ void testRequestsAndValidateAccessLog() throws IOException {
         response = client.get("/access")
                 .header(Http.Header.create(Http.Header.CONTENT_LENGTH, "47a"))
                 .request();
-
         assertThat(response.status(), is(Http.Status.BAD_REQUEST_400));
 
-        try (FileOutputStream fos = new FileOutputStream(ACCESS_LOG.toString(), true)) {
-            fos.getFD().sync();     // force buffer sync
-            List<String> lines = Files.readAllLines(ACCESS_LOG);
-            System.out.println("=== BEGIN access.log ===");
-            lines.forEach(System.out::println);
-            System.out.println("=== END access.log ===");
-            assertThat(lines, contains("127.0.0.1 - [03/Dec/2007:10:15:30 +0000] \"GET /access HTTP/1.1\" 200",
-                    "127.0.0.1 - [03/Dec/2007:10:15:30 +0000] \"GET /wrong HTTP/1.1\" 404"));
+        try {
+            assertThat(LOG_HANDLER.get().contains(
```

Comment:
There is a string contains assertion: 
`assertThat("myString", CoreMatchers.containsString("my"));`
If you use that in the `LOG_HANDLER` instance, you do not need to catch the assertion error, as it will already contain the full output

Reply:
Fixed, but failure in pipeline was a different reason. See changes.